README.md

```diff
@@ -132,6 +107,34 @@ $ docker run -m 512M hazelcast/hazelcast:$HAZELCAST_VERSION
 
 Note that by default Hazelcast uses up to 80% of the container memory limit, but you can configure it by adding `-XX:MaxRAMPercentage` to the `JAVA_OPTS` variable.
 
+### Configuring Hazelcast via Environment Variables
+
+With Hazelcast IMDG 4.1 release, configuration entries of your cluster can be overritten without changing the declarative configuration files (XML/YAML), see [Overriding Configuration documentation section](https://docs.hazelcast.org/docs/latest/manual/html-single/#overriding-configuration).
+
+Assume that you want to have the following configuration for your cluster, represented as YAML:
+```yaml
+hazelcast:
+  cluster-name: dev
+  network:
+    port:
+      auto-increment: true
+      port-count: 100
+      port: 5701
+    join:
+      auto-detection:
+        enabled: true
```

Comment:
```suggestion
hazelcast:
  cluster-name: dev
  network:
    port:
      auto-increment: true
      port-count: 100
      port: 5701
```

Auto-detection is enabled by default, so it does not make much sense to use it in the example.